docs/load-testing.md

```diff
@@ -0,0 +1,43 @@
+# Load testing
+
+We included a sample load testing client in the [../examples/load-tester](../examples/load-tester/) directory. In order to run the test, run the following command:
+
+```
+go run examples/load-tester/main.go -address=<address of the Open Saves server> -connections <number of concurrent connections> \
+    -concurrency <number of concurrent requests per connection> -requests <number of requests> \
+    -tester <name of the load tester>
+```
+
+The following command, for example, invokes the Ping method on a local server using insecure connections (e.g. without TLS).
+
+```
+go run examples/load-tester/main.go -address=localhost:6000 -connections 32 \
```

Comment:
If I followed the deployment guide, what should I put for -address ? I tried this command as is and it failed.

Reply:
That would be the endpoint of the Cloud Run instance, so -address=$ENDPOINT:443. This localhost example was more aligned to the development guide. Do you think we should change it?